pkg/hoist/hoist_launchable.go

```diff
@@ -186,13 +186,13 @@ func (hl *Launchable) stop(serviceBuilder *runit.ServiceBuilder, sv runit.SV) er
 // Start will take a launchable and start every runit service associated with the launchable.
 // All services will attempt to be started.
 func (hl *Launchable) start(serviceBuilder *runit.ServiceBuilder, sv runit.SV) error {
+	var err error
```

Comment:
I guess it's diff noise at this point. In an earlier version I was recycling this value a bit more.